src/client/src/ui/common/popout/popoutOptions.js

```diff
@@ -0,0 +1,25 @@
+export default class PopoutOptions {
+  key:string;
+  url:string;
+  title:string;
+  onClosing:() => void;
+  windowOptions:{
+    width: number,
+    height: number,
+    resizable: string,
+    scrollable: string
+  };
+  constructor(
+    key:string,
+    url:string,
+    title:string,
+    onClosing,
+    windowOptions: {width: number, height: number, resizable: string, scrollable: string}
```

Comment:
`resizeable=no` is not respected/used in `window.open` anymore (deprecated), but it was there previously because OpenFin does.